python_scripts/datasets_adult_census.py

```diff
@@ -0,0 +1,160 @@
+# ---
+# jupyter:
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %% [markdown]
+# # The Adult census dataset
```

Comment:
For consistency with the casing conventions of titles of other dataset sections in the appendix:

```suggestion
# # The adult census dataset
```